### PR TITLE
More cypress

### PR DESCRIPTION
**Pull request by CuriouzK0d3r**
### Description of PR

1. Added more Cypress e2e tests! (change password, admin approves user)
2. Improved the creadentials/ldap login tests
3. Improved our integration with gitpod.io

What component(s) does this PR affect?

  - [x] Back-End (Node.js, Next.js)
  - [ ] Front-End (React.js)
  - [ ] Theming (CSS, React theme)
  - [x] Build System
  - [ ] Docs
  
  
  
  Does the PR require changes on other components? If yes, please mark the components:
  
  - [ ] Back-End (Node.js, Next.js)
  - [ ] Front-End (React.js)
  - [ ] Theming (CSS, React theme)
  - [ ] Build System
  - [ ] Docs
  
  
  
### Related Issue
  
  
  
  
  
  
  Resolves #
  
### Type
  
  - [ ] Bug fix (non-breaking change which fixes an issue)
  - [ ] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to change)
  - [ ] Docs update
  - [ ] None of the above
  
  
  
### Checklist:
  
  - [ ] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
  - [ ] This change requires a change in the documentation.
  - [ ] I have updated the documentation accordingly.
  >)`>>>)>

### CODE REVIEW

File: apps/artemis-web/scripts/query.js [hunk 1]
```diff
@@ -24,6 +26,7 @@ async function isInCollection(db, name) {
       });
     }
     const user = await db.collection('users').findOne();
+    console.log(user)
```
Comment (vkotronis) [inline apps/artemis-web/scripts/query.js:29]: remember to remove this before merging (if used only for testing)

File: apps/artemis-web/components/learn-rule/learn-rule.tsx [hunk 2]
```diff
@@ -145,10 +145,6 @@ class LearnRuleComponent extends Component<
         window[key] = DMP[key];
       });
 
-      const key = this.hijack.key;
-      const prefix: string = this.hijack.prefix;
-      const dashedPrefix = prefix.replace(/\./g, '_').replace('/', '_');
```
Comment (vkotronis) [inline apps/artemis-web/components/learn-rule/learn-rule.tsx:150]: why were these removed?

**Review by vkotronis — COMMENTED** [review]
Please provide a detailed summary of the PR so that it is clear what has been changed and why, since I also see changes in the main modules besides the cypress tests. Just write in the PR what tests were: (i) added, (ii) removed (and why) as well as related changes in the main code.

File: apps/artemis-web/components/learn-rule/learn-rule.tsx (hunk 2 above)
Comment (CuriouzK0d3r) [reply apps/artemis-web/components/learn-rule/learn-rule.tsx:150]: They were not used here :)

**Comment by CuriouzK0d3r** [comment]
> Please provide a detailed summary of the PR so that it is clear what has been changed and why, since I also see changes in the main modules besides the cypress tests. Just write in the PR what tests were: (i) added, (ii) removed (and why) as well as related changes in the main code.

I added a related comment :)

**Review by vkotronis — APPROVED** [review]
LGTM. You can merge the new tests.
